<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaCaptura</class>
 <widget class="QWidget" name="ventanaCaptura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Protocolo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IP fuente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IP destino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tam. maximo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tam. minimo</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>861</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Detener</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar a CSV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reanudar</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>561</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Restablecer Filtros</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Grafico</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGrafica">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Protocolo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IP Fuente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IP Destino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Longitud</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxTipoGrafica">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Pastel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Barras</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
